gettsim/taxes/lohn_st.py

```diff
@@ -177,7 +178,7 @@ def vorsorgepauschale_2005_2010() -> float:
     return out
 
 
-def vorsorg_rv_anteil(eink_st_abzuege_params: dict):
+def vorsorg_rv_anteil(eink_st_abzuege_params: dict, jahr: int):
```

Comment:
Why did you add `jahr` as argument here and where is it handed to the function?

Reply:
I think this is just a mistake of us. We thought that since `piecewise_polynomial` requires `jahr `, it should also be an argument of `vorsorg_rv_anteil`.

Reply:
![grafik](https://user-images.githubusercontent.com/37152510/171230568-79fb2d78-fcbe-4706-9162-bfa3eaf41fe3.png)

What is the right way to call the year as an input for the piecewise_polynomial? 

Before it was `x=pd.Series(eink_st_abzuege_params["datum"].year)`. To apply to the new conventions we replaced it by `x=jahr` which obviously is not working.

Reply:
The parameter `vorsorge_pauschale_rv_anteil` should be handled in `policy_environment.py` similar to e.g. `_parse_einführungsfaktor_vorsorgeaufw_alter_ab_2005`. That way, `eink_st_abzuege_params` would contain `vorsorge_pauschale_rv_anteil` for the correct year as an entry.

Reply:
Should I also change the function name to `_parse_vorsorge_pauschale_rv_anteil`?

Reply:
Could you please [check](https://github.com/iza-institute-of-labor-economics/gettsim/pull/387/commits/1f8e9c6178d1b711a76fa3df67111566f2271709) whether these are the changes you had in mind? I tried to stick to `_parse_einführungsfaktor_vorsorgeaufw_alter_ab_2005` as close as possible.

Reply:
The Parameters section in the docstring is not up-to-date. Otherwise, it looks great!